docs/src/sandu_projection.md

```diff
@@ -0,0 +1,78 @@
+# [Tutorial: Positive-projection method](@id tutorial-sandu)
+
+This tutorial is about solving an ODE using the projection method introduced by Adrian Sandu in [Positive Numerical Integration Methods for Chemical Kinetic Systems](https://doi.org/10.1006%2Fjcph.2001.6750). It guarantees positivity by solving an optimization problem while preserving all linear invariants.
+
+The Sandu projection is a post-processing technique that can be used in combination with any ODE solver.
+If the ODE solver computes a negative approximation at any time step, the projection method calculates a positive approximation, also taking into account the linear invariants.
+
+## Solution of the ODE system
+
+As an example we want to the solve the NPZD problem [`prob_pds_npzd`](@ref), which is an ODE system in which negative approximations quickly lead to unacceptable solutions. First, we solve the problem without Sandu projection and select `ROS2` form [`OrdinaryDiffEq.jl`](https://docs.sciml.ai/OrdinaryDiffEq/stable/) as ODE solver.
+
+```@example Sandu_NPZD
+using PositiveIntegrators
+using OrdinaryDiffEqRosenbrock
+using Plots
+
+prob = prob_pds_npzd
+
+ref_sol = solve(prob, ROS2(); abstol = 1e-8, reltol = 1e-6); # reference solution for plotting
+
+sol = solve(prob, ROS2(); abstol = 5e-2, reltol = 1e-1)
+
+plot(ref_sol, linestyle = :dash, label = "", color = palette(:default)[1:4]')
+plot!(sol, ylims = (-2.5, 12.5), denseplot = false,  markers = :circle, linewidth = 2, color = palette(:default)[1:4]', label = ["N" "P" "Z" "D"], legend = :right)
+```
+
+The plot shows the numerical solution obtained with `ROS2` compared to a reference solution (dashed lines).
+We see that the `ROS2` method produces negative approximations, which can occur because Rosenbrock methods are not positivity-preserving. For the NPZD problem, however, this is fatal and leads to a completely unacceptable numerical solution. It is therefore particularly important to use techniques that guarantee positivity of the numerical approximations for this problem. We achieve this below with the [`SanduProjection`](@ref)
```

Comment:
```suggestion
We see that the `ROS2` method produces negative approximations, which can occur because Rosenbrock methods are not positivity-preserving. For the NPZD problem, however, this is fatal and leads to a completely unacceptable numerical solution. It is therefore particularly important to use techniques that guarantee positivity of the numerical approximations for this problem. We achieve this below with the [`SanduProjection`](@ref).
```